backend/api/Areas/Property/Models/Property/PropertyModel.cs

```diff
@@ -1,12 +1,14 @@
 using System;
 using System.Collections.Generic;
+using Pims.Api.Models;
+using Pims.Api.Models.Concepts;
 
 namespace Pims.Api.Areas.Property.Models.Property
 {
     /// <summary>
     /// PropertyModel class, provides a model to represent the property whether Land or Building.
     /// </summary>
-    public class PropertyModel
+    public class PropertyModel : BaseAppModel
```

Comment:
nit: this one adds the app changes stamps. I dont think we used those. Insead just implementing the interface 
 BaseModel would suffice

Reply:
I can change but see no harm in exposing that.